airflow/contrib/operators/gcp_vision_operator.py

```diff
@@ -0,0 +1,599 @@
+        missing, the default project_id from the GCP connection is used.
+    :type project_id: str
+    :param product_set_id: (Optional) A user-supplied resource id for this ProductSet.
+        If set, the server will attempt to use this value as the resource id. If it is
+        already in use, an error is returned with code ALREADY_EXISTS. Must be at most
+        128 characters long. It cannot contain the character /.
+    :type product_set_id: str
+    :param retry: (Optional) A retry object used to retry requests. If `None` is
+        specified, requests will not be retried.
+    :type retry: google.api_core.retry.Retry
+        https://googleapis.github.io/google-cloud-python/latest/core/retry.html#google.api_core.retry.Retry
+    :param timeout: (Optional) The amount of time, in seconds, to wait for the request to
+        complete. Note that if retry is specified, the timeout applies to each individual
+        attempt.
+    :type timeout: float
+    :param metadata: (Optional) Additional metadata that is provided to the method.
+    :type metadata: Sequence[Tuple[str, str]]
+    :param gcp_conn_id: The connection ID used to connect to Google Cloud Platform.
+    :type gcp_conn_id: str
+    """
+    # [START vision_productset_create_template_fields]
+    template_fields = ('location', 'project_id', 'product_set_id', 'gcp_conn_id')
+    # [END vision_productset_create_template_fields]
+
+    @apply_defaults
+    def __init__(self, product_set,
+                 location,
+                 project_id=None,
+                 product_set_id=None,
+                 retry=None,
+                 timeout=None,
+                 metadata=None,
+                 gcp_conn_id='google_cloud_default',
+                 *args, **kwargs):
+        super(CloudVisionProductSetCreateOperator, self).__init__(*args, **kwargs)
+        self.location = location
+        self.project_id = project_id
+        self.product_set = product_set
+        self.product_set_id = product_set_id
+        self.retry = retry
+        self.timeout = timeout
+        self.metadata = metadata
+        self.gcp_conn_id = gcp_conn_id
+        self._hook = CloudVisionHook(gcp_conn_id=self.gcp_conn_id)
+
+    def execute(self, context):
+        try:
+            return self._hook.create_product_set(location=self.location,
+                                                 project_id=self.project_id,
+                                                 product_set=self.product_set,
+                                                 product_set_id=self.product_set_id,
+                                                 retry=self.retry,
+                                                 timeout=self.timeout,
+                                                 metadata=self.metadata)
+        except AlreadyExists:
+            self.log.info('Product set with id %s already exists. Aborting the create operation.',
```

Comment:
Aborting -> Finishing. It's not an error.

Reply:
Done, changed it to `Exiting from`.